index.html

```diff
@@ -0,0 +1,1704 @@
+                mx-auto">
+                            <div class="position-relative">
+                                <img class="img" src="img/col-1/newsweek-sep19.jpg" alt="" />
+                                <div class="position-absolute category-tag
+                    blue">
+                                    <span class="d-block">SPONSORED INSIGHT</span>
+                                </div>
+                            </div>
+                            <div class="sponsor">
+                                <h3>
+                                    <a class="links" href="#">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                                        Cum, suscipit?</a>
+                                </h3>
+                                <p>
+                                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Eum id obcaecati, eligendi
+                                    accusantium culpa soluta, nemo minima corrupti cum repellat impedit consequuntur
+                                    atque, odio harum!
+                                </p>
+
+                            </div>
+                        </article>
+                    </div>
+                </div>
+
+            </div>
+
+            <div class="container-fluid">
+                <div class="row">
+                    <div class="col-12 col-md-3">
+                        <article class="magazines">
+                            <div class="cover-tag-img">
+
+                                <a href="#"><img class="cover-img"
+                                        src="https://d.newsweek.com/en/full/1548587/december-06-13-2019-cover.webp?w=290&h=387&q=90&f=cddab599081ff086f4af9ee8bbaf747f"
+                                        alt="..." /></a>
+                                <div class="cover-tag ">
+                                    <h2 class="mb-0 "><span class="badge badge-light w-25">OCTOBER 18 ISSUE</span></h2>
+                                </div>
+                            </div>
+
+
+                            <div class="category-tag"><a href="#" class="d-block">SEE ALL FEATURES</a></div>
+
+                        </article>
+                    </div>
+                    <div class="col-12 col-md-9">
+                        <div class="divider w-100 d-flex justify-content-center">
+                            <div class="newsweek-logo-s"></div>
+                            <span>IN THE MAGAZINE</span>
+                        </div>
+                        <div class="row">
+                            <div class="col-12 col-md-4">
+                                <article class="magazine-stories">
+                                    <a href="#"><img class="img-magazine"
+                                            src="https://d.newsweek.com/en/full/1548831/fe-csr-01-1041330598-banner.webp?w=466&h=311&l=52&t=49&q=90&f=e6d8a68693cdcf8906bbbe92a804bc55"
```

Comment:
The "IN THE MAGAZINE" section is messy on a medium screen. [Link to page](https://www.newsweek.com/).

**Yours**
<img width="854" alt="Screenshot 2020-01-03 at 17 15 48" src="https://user-images.githubusercontent.com/31304943/71734755-e86bea00-2e4c-11ea-8bd7-2b68d553c99d.png">

**Original**
<img width="979" alt="Screenshot 2020-01-03 at 17 16 06" src="https://user-images.githubusercontent.com/31304943/71734775-f28de880-2e4c-11ea-87db-08a1f7901f77.png">